### ISSUE TITLE
Alpine 3.12 vulnerabilities are not detected 

### ISSUE THREAD

**Issue by nightfurys** [anchore/anchore-engine#482]
**Is this a request for help?**: No

---

**Is this a BUG REPORT or a FEATURE REQUEST?** (choose one): bug report
Vulnerability listings for alpine 3.12 based images do not yield any results. This is because the feed corresponding to the release is not available from upstream (ancho.re). Alpine secdb has deprecated the source used by the upstream for gathering vulnerability data. Although secdb has recorded vulnerabilities against 3.12, they are not available from the deprecated source

This is just an FYI. No fix necessary in anchore-engine. After the feed data is made available by upstream, anchore-engine will sync this data and vulnerability matches should show up eventually.   

**Version of Anchore Engine and Anchore CLI if applicable**:

**What happened**: Vulnerability listing for an image with alpine 3.12 as the base yielded 0 vulnerabilities

**What did you expect to happen**: Expected anchore-engine to detect vulnerabilities recorded by alpine secdb for 3.12

**Any relevant log output from /var/log/anchore**:

**What docker images are you using**:

**How to reproduce the issue**:

**Anything else we need to know**:

**Comment by bhuisgen** [comment]
Any news on this issue ? The alpine 3.12 feed is still missing here ...

**Comment by zhill** [comment]
The service has been upgraded and 3.12 is now available.

```
curl https://ancho.re/v1/service/feeds/vulnerabilities | grep alpine
"description": "Group record for namespace: alpine:3.10 and feed type: vulnerabilities",
      "name": "alpine:3.10"
      "description": "Group record for namespace: alpine:3.11 and feed type: vulnerabilities",
      "name": "alpine:3.11"
      "description": "Group record for namespace: alpine:3.12 and feed type: vulnerabilities",
      "name": "alpine:3.12"
      "description": "Group record for namespace: alpine:3.2 and feed type: vulnerabilities",
      "name": "alpine:3.2"
      "description": "Group record for namespace: alpine:3.3 and feed type: vulnerabilities",
      "name": "alpine:3.3"
      "description": "Group record for namespace: alpine:3.4 and feed type: vulnerabilities",
      "name": "alpine:3.4"
      "description": "Group record for namespace: alpine:3.5 and feed type: vulnerabilities",
      "name": "alpine:3.5"
      "description": "Group record for namespace: alpine:3.6 and feed type: vulnerabilities",
      "name": "alpine:3.6"
      "description": "Group record for namespace: alpine:3.7 and feed type: vulnerabilities",
      "name": "alpine:3.7"
      "description": "Group record for namespace: alpine:3.8 and feed type: vulnerabilities",
      "name": "alpine:3.8"
      "description": "Group record for namespace: alpine:3.9 and feed type: vulnerabilities",
      "name": "alpine:3.9"
```

**Comment by zhill** [comment]
I'm going to close this issue now that the data is available. Nice work @nightfurys !

**Comment by bhuisgen** [comment]
@zhill thanks but can you automate this to not reproduce this problem at the next alpine release and so on ?

**Comment by zhill** [comment]
@bhuisgen these updates are normally automated. The service detects new versions automatically, but in this case the Alpine team decided to change where the data is hosted with only hours of notice that it would move. The old location used by our service was no longer updated and thus didn't have the 3.12 data. The fix we implemented was to use the new location which is regularly updated and we'll continue to get new versions automatically there just as we did for the old location.
